### PR TITLE
Custom ML Functions

### PR DESCRIPTION
**Pull request by GitDenisGostev**

### CODE REVIEW

**Review by alex-aprm — APPROVED** [review]

**Comment by alex-aprm** [comment]
@konstantin-doncov what do you think?

File: packages/CustomML/src/package.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+export let _package = new DG.Package();
+
+//name: test
+export function test() {
+  grok.shell.info(_package.webRoot);
+}
```
Comment (konstantin-doncov) [inline packages/CustomML/src/package.js:11]: Better to remove this test function

**Review by konstantin-doncov — CHANGES_REQUESTED** [review]
Please remove the test function

File: packages/CustomML/src/package.js (hunk 1 above)
Comment (Denomet) [reply packages/CustomML/src/package.js]: Removed test function from package.js

**Pull request merged**